index.html

```diff
@@ -67,207 +67,262 @@ <h2>LET'S CONNECT</h2>
+          </div>
+          <div class="main-knowledges">
+            <ul class="knowledges">
+              <li class="languages knw">
+                <div class="lang">
+                  <h3>Langguages</h3>
+                  <i class="fas fa-chevron-down"></i>
+                </div>
+                <ul>
+                  <li>
+                    <img src="image/css.svg" class="img-list" alt="javascript"/>
+                    <pre>javaScript</pre>
+                  </li>
+                  <li>
+                    <img src="image/js.svg" class="img-list" alt="html"/>
+                    <pre>html</pre>
+                  </li>
+                  <li>
+                    <img src="image/html.svg" class="img-list" alt="css"/>
+                    <pre>css</pre>
+                  </li>
+                </ul>
               </li>
-              <li>
-                <a href="#">
-                  <img src="image/twitter.png" class="icon" alt="twitter"/>
-                </a>
+              <li class="frameworks knw">
+                <div class="lang">
+                  <h3>Frameworks</h3>
+                  <i class="fas fa-chevron-right"></i>
+                </div>
               </li>
-              <li>
-                <a href="#">
-                  <img src="image/medium.png" class="icon" alt="mediamarket"/>
-                </a>
+              <li class="skills knw">
+                <div class="lang">
+                  <h3>Skills</h3>
+                  <i class="fas fa-chevron-right"></i>
+                </div>
               </li>
             </ul>
-            <div class="btn-T button">
-              <button type="button">Get my resume</button>
-            </div>
           </div>
-          <ul class="knowledges">
-            <li class="languages knw">
-              <div class="lang">
-                <h3>Langguages</h3>
-                <i class="fas fa-chevron-down"></i>
-              </div>
-              <ul>
-                <li>
-                  <img src="image/css.svg" class="img-list" alt="javascript"/>
-                  <h4>javaScript</h4>
-                </li>
-                <li>
-                  <img src="image/js.svg" class="img-list" alt="html"/>
-                  <h4>html</h4>
-                </li>
-                <li>
-                  <img src="image/html.svg" class="img-list" alt="css"/>
-                  <h4>css</h4>
-                </li>
-              </ul>
-            </li>
-            <li class="frameworks knw">
-              <div class="lang lang-b">
-                <h3>Frameworks</h3>
-                <i class="fas fa-chevron-right"></i>
-              </div>
-            </li>
-            <li class="skills knw">
-              <div class="lang lang-b">
-                <h3>Skills</h3>
-                <i class="fas fa-chevron-right"></i>
-              </div>
-            </li>
-          </ul>
         </div>
       </section>
       <section class="contact-section">
```

Comment:
Fix the background styling of the contact form
### Original design
![right](https://user-images.githubusercontent.com/58376020/126869283-e1f818c1-9ae0-410c-9467-eec86e5eb95e.png)

### Your Project
![wrong](https://user-images.githubusercontent.com/58376020/126869289-aaf1c7e0-e640-4abb-b289-2a2d9017e1a5.png)